### PR TITLE
US 1 - Play-Card 

### PR DESCRIPTION
**Pull request by milodevhh**
-  add lib folder, add a db.json / db.js with an Array of 9 objects with the Key/value e.g.
-  create a card component add function with map over Array
-  add a container to render the div, add a for each card, add a hover,
-  add an on click function - change color #F2E7AE - toggle
-  Return it in index.js
-  add a first styling (Grid, Icon size, Text size)

### CODE REVIEW

File: components/Card.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import initialCards from "@/lib/db";
+import { useState } from "react";
+import styled from "styled-components";
+
+export default function Card({}) {
```
Comment (F-Kirchhoff) [inline components/Card.js]: you can remove the curly brackets

File: components/Card.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      } else {
+        const updatedCard = {
+          ...card,
+          isActive: !card.isActive,
+        };
+        return updatedCard;
+      }
+    });
+    setCards(updatedCards);
+    console.log(updatedCards);
```
Comment (F-Kirchhoff) [inline components/Card.js]: please remove console logs 🕵️

File: components/Card.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+            key={card.id}
+            type="button"
+            onClick={() => handleActive(card.id)}
+            $isActive={card.isActive}
+          >
+            {card.name} {card.icon}
+          </GameCard>
+        ))}
+      </GridContainer>
+      <p>You can add your own Playcards later on.</p>
```
Comment (F-Kirchhoff) [inline components/Card.js]: you don't need this statement.

File: components/Card.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    </>
+  );
+}
+
+const GridContainer = styled.div`
```
Comment (F-Kirchhoff) [inline components/Card.js]: please use ul and li elements when mapping over an array (ul as container, li as children)

File: lib/db.json [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+  },
+  {
+    "icon": "🛝",
+    "name": "Slide"
+  },
+  {
+    "icon": "🍓",
+    "name": "Strawberry"
+  }
+]
```
Comment (F-Kirchhoff) [inline lib/db.json:38]: why do you have a db.json and db.js?

File: components/Card.js (hunk 1 above)
Comment (F-Kirchhoff) [inline components/Card.js]: this component displays a List of cards. please rename it accordingly. At the moment you might think that it renders only one card.

File: pages/index.js [hunk 6]
```diff
@@ -1,7 +1,14 @@
+import Card from "@/components/Card";
+
 export default function HomePage() {
   return (
     <div>
-      <h1>Hello from Next.js</h1>
+      <h1>Bingo</h1>
+      <h2>Play</h2>
+
+      <ul>
```
Comment (F-Kirchhoff) [inline pages/index.js]: when you use the ul in the component as suggested you don't need it here any longer.

File: pages/index.js [hunk 7]
```diff
@@ -1,7 +1,14 @@
+import Card from "@/components/Card";
+
 export default function HomePage() {
   return (
     <div>
```
Comment (F-Kirchhoff) [inline pages/index.js]: I would suggest to use semantic HTML instead of divs, for example a main tag

**Review by F-Kirchhoff — CHANGES_REQUESTED** [review]
Hey Milo, looking great already. Have a look at my comments.

File: components/Card.js (hunk 4 above)
Comment (milodevhh) [reply components/Card.js]: Thanks, add styling in GameCard

File: lib/db.json (hunk 5 above)
Comment (milodevhh) [reply lib/db.json:38]: I Want to use MongoDB Atlas later on, so I have a default .json to setup the collection.

File: components/Card.js (hunk 1 above)
Comment (milodevhh) [reply components/Card.js]: I think GridOfCards is good, or?

File: components/Card.js [hunk 8]
```diff
@@ -47,13 +45,13 @@ const GridContainer = styled.div`
   padding: 2rem;
 `;
 
-const GameCard = styled.button`
+const GameCard = styled.li`
```
Comment (F-Kirchhoff) [inline components/Card.js]: if your Card is clickable it has to be a button. Please put a button inside a li element (you might to style both, depending on your case).

**Review by F-Kirchhoff — CHANGES_REQUESTED** [review]
You are almost done! Please add the one suggestion, then you are good to go.

**Review by F-Kirchhoff — APPROVED** [review]
LGTM 🦊

**Pull request merged**